src/prevector.h

```diff
@@ -220,7 +220,10 @@ class prevector {
         }
     }
 
-    prevector() : _size(0) {}
+    prevector() : _size(0) {
+        // Explicitly initialize indirect ptr to avoid "used uninitialized" warning
+        _union.indirect = nullptr;
```

Comment:
From IRC:
<cfields> sipa: why not use aggregate initialization: prevector() : _size(0), _union{{}}
<cfields> or give the union a ctor?

I can't reproduce the warning, so I can't be sure that it satisfies whatever compiler is complaining, but the above compiles fine for me, and I'd think that initializing via a ctor would work otherwise.